### ISSUE TITLE
[bug]: No button for adapter update visible

### ISSUE THREAD

**Issue by Feuer-sturm** [ioBroker/ioBroker.admin#2498]
### No existing issues.

- [X] There is no existing issue for my problem.

### Describe the bug

An update of the adapter is availbale but there is no button visible to start the update
![grafik](https://github.com/ioBroker/ioBroker.admin/assets/25422320/dbd87fd7-f6e1-4294-b105-8385dc556a5b)

### To Reproduce

Try to update an adapter with admin 7.0.1

### Expected behavior

Button is visible

### Screenshots & Logfiles
Just to check if update is available on CLI

```
proxmox@ioBrokerTestsystem:~$ iob update
Used repository: beta
Adapter    "admin"        : 7.0.1    , installed 7.0.1
Adapter    "backitup"     : 3.0.3    , installed 3.0.3
Adapter    "benchmark"    : 1.3.0    , installed 1.3.0
Adapter    "daswetter"    : 3.1.15   , installed 3.1.15
Adapter    "device-watcher": 2.12.1  , installed 2.12.1
Adapter    "devices"      : 1.1.5    , installed 1.1.5
Adapter    "discovery"    : 4.5.0    , installed 4.5.0
Adapter    "dwd"          : 2.8.5    , installed 2.8.5
Adapter    "echarts"      : 1.7.2    , installed 1.7.2
Adapter    "fritzbox"     : 0.6.0    , installed 0.6.0
Adapter    "hm-rega"      : 4.0.0    , installed 4.0.0
Adapter    "hm-rpc"       : 1.17.0   , installed 1.17.0
Adapter    "hue"          : 3.11.0   , installed 3.11.0
Adapter    "ical"         : 1.15.0   , installed 1.15.0
Adapter    "icons-material-png": 0.1.0, installed 0.1.0
Adapter    "icons-mfd-svg": 1.2.0    , installed 1.2.0
Adapter    "icons-ultimate-png": 1.0.1, installed 1.0.1
Adapter    "influxdb"     : 4.0.2    , installed 4.0.2
Adapter    "javascript"   : 8.5.1    , installed 8.5.1
Controller "js-controller": 6.0.1    , installed 6.0.1
Adapter    "mqtt"         : 5.2.0    , installed 5.2.0
Adapter    "nina"         : 0.0.26   , installed 0.0.26
Adapter    "openweathermap": 1.0.4   , installed 1.0.4
Adapter    "proxmox"      : 2.3.0    , installed 2.3.0
Adapter    "radar-trap"   : 2.2.0    , installed 2.2.0
Adapter    "rainbird"     : 1.0.1    , installed 1.0.1
Adapter    "samsung"      : 0.6.0    , installed 0.6.0
Adapter    "sayit"        : 4.0.1    , installed 4.0.1
Adapter    "scenes"       : 3.0.4    , installed 3.0.4
Adapter    "scheduler"    : 1.3.12   , installed 1.3.12
Adapter    "simple-api"   : 2.8.0    , installed 2.8.0
Adapter    "socketio"     : 6.7.0    , installed 6.7.0
Adapter    "spotify-premium": 1.5.0  , installed 1.5.0
Adapter    "tankerkoenig" : 3.4.0    , installed 3.4.0
Adapter    "telegram"     : 3.4.1    , installed 3.3.2  [Updatable]
Adapter    "tr-064"       : 4.3.0    , installed 4.3.0
Adapter    "trashschedule": 3.3.0    , installed 3.3.0
Adapter    "unifi"        : 0.7.0    , installed 0.7.0
Adapter    "valloxmv"     : 1.2.0    , installed 1.2.0
Adapter    "vis"          : 1.5.6    , installed 1.5.6
Adapter    "vis-2"        : 2.9.64   , installed 2.9.64
Adapter    "vis-2-widgets-energy": 0.3.11, installed 0.3.11
Adapter    "vis-2-widgets-gauges": 1.0.3, installed 1.0.3
Adapter    "vis-2-widgets-jaeger-design": 1.1.27, installed 1.1.27
Adapter    "vis-2-widgets-material": 1.3.33, installed 1.3.33
Adapter    "vis-fancyswitch": 1.1.0  , installed 1.1.0
Adapter    "vis-google-fonts": 1.0.4 , installed 1.0.4
Adapter    "vis-history"  : 1.0.0    , installed 1.0.0
Adapter    "vis-inventwo" : 3.3.4    , installed 3.3.4
Adapter    "vis-jqui-mfd" : 1.1.1    , installed 1.1.1
Adapter    "vis-justgage" : 2.1.7    , installed 2.1.7
Adapter    "vis-map"      : 1.1.0    , installed 1.1.0
Adapter    "vis-materialdesign": 0.5.9, installed 0.5.9
Adapter    "vis-timeandweather": 1.2.2, installed 1.2.2
Adapter    "web"          : 6.2.6    , installed 6.2.6
Adapter    "worx"         : 3.0.2    , installed 3.0.2
Adapter    "ws"           : 2.6.1    , installed 2.6.1

```

### Adapter version

7.0.1

### js-controller version

6.0.1

### Node version

v18.20.3

### Operating system

ubuntu

**Comment by GermanBluefox** [comment]
Retest with 7.0.2

**Comment by Feuer-sturm** [comment]
@GermanBluefox Installed Admin 7.0.2
After admin upgrade I made a downgrade of two adapters to test the issue but it looks like this:
![grafik](https://github.com/ioBroker/ioBroker.admin/assets/25422320/a5137e30-b4be-4cbf-a7ad-f473b095ba49)

No update button is available. It is necessary to make a Page Reload F5 and then the button is visible:

![grafik](https://github.com/ioBroker/ioBroker.admin/assets/25422320/4b48a7a6-0f41-479e-ba23-8d239fe6362c)

**Comment by Feuer-sturm** [comment]
Problem is still there with admin 7.0.7
After updating socket.io adapter admin and confirmation it looks like this. Update available but no button.

![grafik](https://github.com/ioBroker/ioBroker.admin/assets/25422320/83d31f95-e07f-434e-9e49-2d98b4751278)

After page reload button are available.

**Comment by GermanBluefox** [comment]
Ok. This will be fixed in 7.0.10

**Comment by Feuer-sturm** [comment]
is fixed with admin 7.0.11
